app/points/calculate-sdr-conversion-points.js

```diff
@@ -0,0 +1,3 @@
+module.exports = function (SDRConversionsLast30Days, WorkloadPointsSDRConversion) {
```

Comment:
At the risk of bikeshedding (https://en.wiktionary.org/wiki/bikeshedding), can we use camel case for parameter names as is our custom.